doc/FAQ.md

```diff
@@ -226,25 +227,28 @@ However many people prefer the cleaner aesthetic of no trailing slashes. This co
 to the base path as you cannot use relative redirects correctly anymore. See the above
 link.
 
-For users who are ok with this tradeoff, pass `--proxy-path-passthrough` to code-server
-and the path will be passed as is.
+For users who are ok with this tradeoff, use `/absproxy` instead and the path will be
+passed as is. e.g. `/absproxy/3000/my-app-path`
 
-This is particularly a problem with the `start` script in create-react-app. See
+### Proxying to Create React App
```

Comment:
Good idea calling out "Create React App" - I could see us updating the docs with other particular apps down the road.